Readme.md

```diff
@@ -3,5 +3,7 @@
 ## How to run
 1. Prerequisites:
    - Install rust using rustup
+   - To run the TailwindCSS in watch mode `npx tailwindcss -i ./src/style.css -o ./dist/output.css --watch`
```

Comment:
do we have to run this command along with cargo watch/run too?

Reply:
yes. it watches changes in css and compile into CSS